yb-voyager/src/namereg/namereg.go

```diff
@@ -374,7 +374,8 @@ func (reg *NameRegistry) lookupSourceAndTargetTableNames(tableNameArg string, ig
 				}
 			}
 			if err != nil {
-				if ignoreIfOtherSideMappingNotFound {
+				errNotFound := &ErrNameNotFound{}
+				if ignoreIfOtherSideMappingNotFound && errors.As(err, &errNotFound) {
```

Comment:
`errNotFound` is already a pointer, do you need to pass `&errNotFound` again?

Reply:
Yes, this is how its docs also [says](https://pkg.go.dev/errors#As:~:text=As%20panics%20if%20target%20is%20not%20a%20non%2Dnil%20pointer%20to%20either%20a%20type%20that%20implements%20error%2C%20or%20to%20any%20interface%20type.), this `errors.As` function panics if we pass doesn't pass it like this.